### PR TITLE
Check the latest Version in the signing API

### LINKED ISSUE #953: Handle multiple versions in signing API
https://sentry.prod.mozaws.net/operations/olympia-prod/group/153985/

There can be multiple versions for the same add-on version string if the add-on goes into the listed queue and a new file is uploaded before it is reviewed.

We should check the editor tools to see how the "active" version is selected.

### PR DESCRIPTION
**Pull request by mstriemer**
Fixes #953

### CODE REVIEW

File: apps/signing/tests/test_views.py [hunk 1]
```diff
@@ -104,6 +104,16 @@ def test_version_already_exists(self):
         assert response.status_code == 409
         assert response.data['error'] == 'Version already exists.'
 
+    @mock.patch('devhub.views.Version.from_upload')
+    def test_no_version_yet(self, from_upload):
+        response = self.put(self.url(self.guid, '3.0'))
+        assert response.status_code == 202
+        assert 'processed' in response.data
+
+        response = self.get(self.url(self.guid, '3.0'))
```
Comment (kumar303) [inline apps/signing/tests/test_views.py]: it's not totally obvious from the test name why this is here. Ideally, I like to see things such as this in a separate test function but at the least I think you could just add a comment to explain what it's testing for.

File: apps/signing/tests/test_views.py [hunk 2]
```diff
@@ -141,6 +151,10 @@ def test_version_failed_review(self):
         assert response.status_code == 202
         assert 'processed' in response.data
 
+        response = self.get(self.url(self.guid, '3.0'))
```
Comment (kumar303) [inline apps/signing/tests/test_views.py]: it's not totally obvious from the test name why this is here. Ideally, I like to see things such as this in a separate test function but at the least I think you could just add a comment to explain what it's testing for.

**Comment by kumar303** [comment]
r+wc

cat-lasers

**Comment by mstriemer** [comment]
Fixed in https://github.com/mozilla/olympia/commit/6cc28e1dc8d58a4279f589a4206c3b66900570d5.

**Pull request closed without merging**